vmdb/app/controllers/miq_ae_class_controller.rb

```diff
@@ -825,24 +825,6 @@ def form_instance_field_changed
     get_instances_form_vars
 
     render :update do |page|                    # Use JS to update the display
-
-      params.each do |var, val|
-        # for instance tab on class screen
-        if var.starts_with?("cls_inst_value") || var.starts_with?("cls_inst_password_value") ||
-            var.starts_with?("cls_inst_collect") || var.starts_with?("cls_inst_on_entry") ||
-            var.starts_with?("cls_inst_on_exit") || var.starts_with?("cls_inst_on_error") ||
-            var.starts_with?("cls_inst_max_retries") || var.starts_with?("cls_inst_max_time")
-          @ae_class.ae_fields.sort_by! { |f| f.priority.to_i } if @ae_class
-        end
-        # for instance node selected in the left tree
-        if var.starts_with?("inst_value") || var.starts_with?("inst_password_value") ||
-            var.starts_with?("inst_collect") || var.starts_with?("inst_on_entry") ||
-            var.starts_with?("inst_on_exit") || var.starts_with?("inst_on_error") ||
-            var.starts_with?("inst_max_retries") || var.starts_with?("inst_max_time")
-          @ae_class.ae_fields.sort_by! { |f| f.priority.to_i } if @ae_class
-        end
-      end
-
```

Comment:
This is fantastic!  So many conditionals gone!!!  Yay, thanks @h-kataria!  I :heart: it.  More please :wink: